src/jaxsim/rbda/contacts/rigid.py

```diff
@@ -0,0 +1,479 @@
+from __future__ import annotations
+
+import dataclasses
+from typing import Any
+
+import jax
+import jax.numpy as jnp
+import jax_dataclasses
+
+import jaxsim.api as js
+import jaxsim.typing as jtp
+from jaxsim import math
+from jaxsim.api.common import ModelDataWithVelocityRepresentation, VelRepr
+from jaxsim.terrain.terrain import FlatTerrain, Terrain
+
+from .common import ContactModel, ContactsParams, ContactsState
+
+
+@jax_dataclasses.pytree_dataclass
+class RigidContactParams(ContactsParams):
```

Comment:
Sorry for not having noticed this before
```suggestion
class RigidContactsParams(ContactsParams):
```